components/ww3/src/cpl/ww3_cpl_indices.f90

```diff
@@ -18,9 +18,20 @@ module ww3_cpl_indices
   integer :: index_x2w_So_bldepth     
   integer :: index_x2w_So_ssh
 
-  integer :: index_w2x_Sw_ustokes
-  integer :: index_w2x_Sw_vstokes
-  integer :: index_w2x_Sw_hstokes
+  integer :: index_w2x_Sw_ustokes_wavenumber_1
```

Comment:
mostly a curiosity that could have been anywhere, what does `Sw` stand for throughout the code?

Reply:
thanks @philipwjones good to know!